spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialShareConvolution.scala

```diff
@@ -220,7 +219,8 @@ class SpatialShareConvolution[T: ClassTag](
           gradWeightMM.select(1, g + 1),
           gradBias.narrow(1, g * nOutputPlane / nGroup + 1, nOutputPlane / nGroup),
           fInput.select(1, g + 1),
-          ev.fromType[Double](scale))
+          ev.fromType[Double](scaleW),
```

Comment:
This layer calls the `accGradParametersFrame` in `SpatialConvolution.scala`, in which the scale has been checked.